include/utils.h

```diff
@@ -118,69 +118,67 @@ error::Error CalcPartInfo(long object_size, size_t& part_size,
                           long& part_count);
 
 /**
- * Time represents date and time with timezone.
+ * UtcTime represents date and time in UTC timezone.
  */
-class Time {
+class UtcTime {
  private:
-  struct timeval tv_ = {};
```

Comment:
Thanx!

Reply:
You will find many such things. We will fix them in upcoming PRs. If possible you could open Github issues so that we won't miss out.